diff --color -ur --no-dereference steamdeck-dsp-0.57/pipewire-confs/hardware-profiles/valve-galileo/pipewire.conf.d/filter-chain.conf steamdeck-dsp-0.57-replaced/pipewire-confs/hardware-profiles/valve-galileo/pipewire.conf.d/filter-chain.conf
--- steamdeck-dsp-0.57/pipewire-confs/hardware-profiles/valve-galileo/pipewire.conf.d/filter-chain.conf	2024-11-22 12:20:41.000000000 -0500
+++ steamdeck-dsp-0.57-replaced/pipewire-confs/hardware-profiles/valve-galileo/pipewire.conf.d/filter-chain.conf	2025-01-16 10:55:20.764566963 -0500
@@ -11,16 +11,16 @@
             media.name =  "Filter Chain Source"
             filter.graph = {
                 nodes = [
-                    {
-                        type = ladspa
-                        name = rnnoise
-                        plugin = /usr/lib/ladspa/librnnoise_ladspa.so
-                        label = noise_suppressor_stereo
-                        control = {
-                            "VAD Threshold (%)" = 23.0
-                            "VAD Grace Period (ms)" = 200
-                        }
-                    }
+                    # {
+                    #     type = ladspa
+                    #     name = rnnoise
+                    #     plugin = /usr/lib/ladspa/librnnoise_ladspa.so
+                    #     label = noise_suppressor_stereo
+                    #     control = {
+                    #         "VAD Threshold (%)" = 23.0
+                    #         "VAD Grace Period (ms)" = 200
+                    #     }
+                    # }
                     {
                         type = lv2
                         name = valve_deck_microphone
@@ -29,8 +29,8 @@
                     }
                 ]
                 links = [
-                    { output = "rnnoise:Output (L)" input = "valve_deck_microphone:in0" }
-                    { output = "rnnoise:Output (R)" input = "valve_deck_microphone:in1" }
+                    # { output = "rnnoise:Output (L)" input = "valve_deck_microphone:in0" }
+                    # { output = "rnnoise:Output (R)" input = "valve_deck_microphone:in1" }
                 ]
             }
             capture.props = {
diff --color -ur --no-dereference steamdeck-dsp-0.57/pipewire-confs/hardware-profiles/valve-jupiter/pipewire.conf.d/filter-chain.conf steamdeck-dsp-0.57-replaced/pipewire-confs/hardware-profiles/valve-jupiter/pipewire.conf.d/filter-chain.conf
--- steamdeck-dsp-0.57/pipewire-confs/hardware-profiles/valve-jupiter/pipewire.conf.d/filter-chain.conf	2024-11-22 12:20:41.000000000 -0500
+++ steamdeck-dsp-0.57-replaced/pipewire-confs/hardware-profiles/valve-jupiter/pipewire.conf.d/filter-chain.conf	2025-01-16 10:55:31.771178306 -0500
@@ -11,16 +11,16 @@
             media.name =  "Filter Chain Source"
             filter.graph = {
                 nodes = [
-                    {
-                        type = ladspa
-                        name = rnnoise
-                        plugin = /usr/lib/ladspa/librnnoise_ladspa.so
-                        label = noise_suppressor_stereo
-                        control = {
-                            "VAD Threshold (%)" = 23.0
-                            "VAD Grace Period (ms)" = 200
-                        }
-                    }
+                    # {
+                    #     type = ladspa
+                    #     name = rnnoise
+                    #     plugin = /usr/lib/ladspa/librnnoise_ladspa.so
+                    #     label = noise_suppressor_stereo
+                    #     control = {
+                    #         "VAD Threshold (%)" = 23.0
+                    #         "VAD Grace Period (ms)" = 200
+                    #     }
+                    # }
                     {
                         type = lv2
                         name = valve_deck_microphone
@@ -29,8 +29,8 @@
                     }
                 ]
                 links = [
-                    { output = "rnnoise:Output (L)" input = "valve_deck_microphone:in0" }
-                    { output = "rnnoise:Output (R)" input = "valve_deck_microphone:in1" }
+                    # { output = "rnnoise:Output (L)" input = "valve_deck_microphone:in0" }
+                    # { output = "rnnoise:Output (R)" input = "valve_deck_microphone:in1" }
                 ]
             }
             capture.props = {
